components/WPAtlasTable.tsx

```diff
@@ -31,7 +31,7 @@ export const WPAtlasTable: React.FunctionComponent<IWPAtlasTableProps> = (
         },
         {
             name: 'Atlas Description',
-            selector: 'title.rendered',
+            selector: 'WPAtlas.title.rendered',
```

Comment:
good catch! we should probably use function selectors for nested properties to avoid these kind of errors in the future.

Reply:
good point @onursumer